Makefile

```
@@ -191,6 +191,10 @@ install-ingest-airtable:
 install-ingest-sharepoint:
 	python3 -m pip install -r requirements/ingest-sharepoint.txt
 
+.PHONY: install-ingest-weaviate
+install-ingest-weaviate:
+	python3 -m pip install -r requirements/ingest/ingest-weaviate.txt
```

```suggestion
	python3 -m pip install -r requirements/ingest/weaviate.txt
```
Probably don't need the prepend when the file is in the ingest folder